CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - New cockroach_version_deferral resource allows users to defer automated minor version
   upgrades by a fix 60-day interval.
+- Allowlists and AWS PrivateLink services are now supported for serverless clusters.
```

Comment:
One thing to note is that AWS PrivateLink is guarded behind a feature flag, so it will only work on some orgs/clusters. Not sure if we should clarify it there.

Reply:
Good callout. Added that info and linked to the feature availability doc.